src/clr-addons/clr-addons.module.ts

```diff
@@ -6,10 +6,11 @@
 
 import { NgModule } from '@angular/core';
 
-import { ClrViewEditSectionModule } from './view-edit-section/view-edit-section.module';
```

Comment:
Is there a reason for removing the filename?

Reply:
intelliJ improvement suggestion

Reply:
just wondered because its not consistently removed everywhere e.g. one line below or in the src/dev files